content/community/jdev-rules.md

```diff
@@ -0,0 +1,83 @@
+---
+Title: jdev Channel
+Url: community/channels/jdev
+---
+
+## About the XMPP Developers Channel ("jdev")
+
+The XMPP Developers Channel is located at [jdev@muc.xmpp.org](xmpp:jdev@muc.xmpp.org?join). It is a place primarily for developers of software following or aiming to follow XMPP standards (RFC 6120--6122, as well as XMPP Extension Proposals, and updated versions of the RFCs), to have civil and low-bar knowledge transfer, learning and support in the task of implementing the various pieces of the XMPP standard compendium.
+
+## Code of Conduct
+
+Like any growing community, the XMPP Developers Channel has a set of rules to govern the interactions, in order to make as many people as possible feel safe and welcome.
+
+Above all, the [Community Code of Conduct of the XMPP Standards Foundation](https://xmpp.org/extensions/xep-0458.html) (still under discussion, but unless stated otherwise in *this* document, the current experimental version applies nontheless) applies, called CoC hereafter. That means that behaviour which goes against the CoC is automatically not accepted.
+
+For transgressions solely happening within the XMPP Developers channel, the channel admins are an acceptable first point of contact.
+
+**Note**: The CoC only applies to XMPP Standards Foundation (XSF) members and venues. It does not automatically apply to all members of all XMPP channels. Do not report misconduct in venues outside of the control of the XSF to this room or the XSF Conduct Team, unless an XSF member is involved.
```

Comment:
Maybe a topic to explore elsewhere, but does the CoC apply to *any* XSF member *anywhere*? Or just when they act in quality of XSF member?